docs/markdown/Syntax.md

```diff
@@ -560,6 +560,19 @@ The only exception is that nested ternary operators are forbidden to
 improve legibility. If your branching needs are more complex than this
 you need to write an `if/else` construct.
 
+Null-Coalescing operator
+--
+
+The null-coalescing operator works just like in C# language.
```

Comment:
Which is what exactly? Now I need to go find a C# tutorial instead of it being explained here.